kubernetes-model/kubernetes-model-annotator/src/main/java/io/fabric8/kubernetes/annotator/KubernetesCoreTypeAnnotator.java

```diff
@@ -102,6 +104,26 @@ public void propertyOrder(JDefinedClass clazz, JsonNode propertiesNode) {
     }
   }
 
+  private boolean isResourceNamespaced(JDefinedClass resourceClass) {
+    switch (resourceClass.name()) {
```

Comment:
if we can avoid hardcoding strings here and [here](https://github.com/fabric8io/kubernetes-client/blob/29c98553e599abc9e5751acddcef59daf6bdcc41/kubernetes-model/kubernetes-model-annotator/src/main/java/io/fabric8/kubernetes/annotator/KubernetesCoreTypeAnnotator.java#L193), that would be great. Otherwise everything looks good to me.